app/scripts/lib/app-start.js

```diff
@@ -60,6 +60,7 @@ define([
   'models/auth_brokers/iframe',
   'models/unique-user-id',
   'models/user',
+  'models/verification/same-browser',
```

Comment:
One idea I was tossing around is a universal "verification info model". We have some info persisted in localStorage, some from query parameters, and some in the resume token. Can we combine all those sources of information into a single, unified interface?